src/scripts/components/stories/story/blocks/image-gallery/formats/scroll-caption/scroll-caption.tsx

```diff
@@ -14,9 +15,14 @@ const ScrollCaption: FunctionComponent<StorySectionProps> = ({ ref }) => {
     <FormatContainer ref={ref} className={styles.scrollCaptionBlock}>
       {content.slides.map((slide, index) => (
         <div key={index} className={styles.scrollImageContainer}>
-          <CaptionImage src={getStoryAssetUrl(storyId, slide.url)} alt={slide.altText} />
-          <h3>{slide.caption}</h3>
-          <p>{slide.altText}</p>
+          <CaptionImage
+            src={getStoryAssetUrl(storyId, slide.url)}
+            alt={slide.altText}
+          />
+          <ReactMarkdown
+            children={slide.captions.join("\n\n")}
+            allowedElements={["h3", "p", "br", "em", "b", "a"]}
```

Comment:
Should we really restrict the `allowedElements`? What happens, if other elements are defined in captions?